engine/flink/management/sample/src/test/resources/extractedTypes/devCreator.json

```diff
@@ -10027,40 +10113,6 @@
       ]
     }
   },
-  {
-    "clazzName": {"refClazzName": "pl.touk.nussknacker.engine.management.sample.dto.CsvRecord"},
```

Comment:
Interesting why it was here and after removal still works. We have some bug?

Reply:
I'm assuming that at some point we have removed CsvRecord from supported Classes, and this file was not updated.